.github/workflows/kata-deploy-push.yaml

```diff
@@ -56,3 +56,14 @@ jobs:
         with:
           name: kata-static-tarball
           path: kata-static.tar.xz
+
+  make-kata-tarball:
+    runs-on: ubuntu-latest
```

Comment:
Maybe also Fedora?

Reply:
Not officially, no.  In order to make it run "out-of-the-box" on Fedora we should make it work with podman, as right now it's quite docker specific.

In order to avoid false expectations (and then issues coming up that we won't work on) I'd prefer to not advertise it works there.

Does this sound reasonable?

Reply:
I agree with @fidencio additionally even if supported, `runs-on` from github actions only supports ubuntu, in the future if we want to test in more places we would need to use jenkins for somthing like that, the advantage is that even we test only in ubuntu, it is likely that having a linux system with docker will work (I hope  :P ) .